doc/configuration-v3_0.md

```diff
@@ -69,7 +69,7 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_group_** (object): specifies the group of the owner.
       * **_id_** (integer): the group ID of the owner.
       * **_name_** (string): the group name of the owner.
-  * **_directories_** (list of objects): the list of directories to be created.
+  * **_directories_** (list of objects): the list of directories to be created. Every file, directory, and link must have a unique path.
```

Comment:
Files wraps `path` in back-ticks while directories & links do not. Should probably update both directories & links to wrap them (as well as `name` in systemd units, drop-ins, users, & groups) for consistency.